CMakeLists.txt

```diff
@@ -0,0 +1,201 @@
+cmake_minimum_required(VERSION 3.8)
+
+project(usd-unity-sdk LANGUAGES CXX)
+set (CMAKE_CXX_STANDARD 11)
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
+
+find_package(pxr REQUIRED)
+find_package(SWIG 3.0.12 REQUIRED)
+find_package(Boost REQUIRED)
```

Comment:
@cpredmann Does this find the package installed on the OS ? If yes, I think we should control much more tightly the dependency versions to be able to be more reproducible (download any dependencies as a part of the built process)

Reply:
Yes, it'll just look for those modules, but you're right that it doesn't (aside from SWIG) do any version checking.  I could add ExternalProject_Add steps for the dependencies to make sure it's all built correctly (version-wise).  That would also solve the problem of potentially dynamically linking against random versions of dependencies, if they were built/installed as part of the process.

Reply:
+1, versions are specific and should stay in lock step with the USD build script dependency versions (which themselves are driven by the VFX Platform).

Reply:
Yes, the updated version of the CMake builds with the specific version dependencies, to avoid any compilation issues/conflicts.